packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -9,7 +9,7 @@ import PropTypes from 'prop-types';
 
 export interface AccordionTriggerProps
   extends React.HTMLAttributes<HTMLButtonElement> {
-  children: React.ReactElement;
+  children: React.ReactNode;
```

Comment:
Trigger was not accepting a standalone `string` (e.g., `<AccordionTrigger>Trigger title</AccordionTrigger>`), as I believe `React.ReactElement` is expecting a JSX element whereas `ReactNode` is more inclusive and can be a `string`.